.github/workflows/knative-releasability.yaml

```diff
@@ -19,35 +19,63 @@ name: 'Releasability'
 
 on:
   schedule:
-    - cron: '0 5 * * 1-5' # 5am, robot time, weekdays.
+    - cron: '0 1 * * 1-5' # 6am Pacific, weekdays.
 
-  workflow_dispatch:    # Manual trigger.
+  workflow_dispatch:      # Manual trigger.
+    inputs:
+      releaseFamily:
+        description: 'Release? (vX.Y)'
+      slackChannel:
+        description: 'Slack Channel? (release-#)'
 
 jobs:
   releasability:
     name: Releasability
     runs-on: 'ubuntu-latest'
 
     env:
-  #########################################
-  #   Update this section each release.   #
-      RELEASE: v0.19
-      SLACK_CHANNEL: 'mattmoor-slack-test'
-  #########################################
+      #########################################
+      #   Update this section each release.   #
+      RELEASE: 'v0.18'
+      SLACK_CHANNEL: 'release-18'
+      #########################################
 
     steps:
+      - name: Defaults
+        run: |
+          # If manual trigger sent releaseFamily and slackChannel, then override them
+          if [[ "${{ github.event.inputs.releaseFamily }}" != "" ]]; then
+            echo "RELEASE=${{ github.event.inputs.releaseFamily }}" >> $GITHUB_ENV
+          fi
+          if [[ "${{ github.event.inputs.slackChannel }}" != "" ]]; then
+            echo "SLACK_CHANNEL=${{ github.event.inputs.slackChannel }}" >> $GITHUB_ENV
+          fi
+
       - name: Set up Go 1.15.x
         uses: actions/setup-go@v2
         with:
           go-version: 1.15.x
 
       - name: Install Dependencies
-        run: GO111MODULE=on go get tableflip.dev/buoy@master
+        run: GO111MODULE=on go get knative.dev/test-infra/buoy@master
 
       - name: Check out code
         uses: actions/checkout@v2
 
+      - name: Exists
+        id: exists
+        run: |
+          EXISTS=0
+          buoy exists go.mod --release ${RELEASE} --verbose || EXISTS=$?
+          if [[ "$EXISTS" -eq "0" ]]; then
+            EXISTS=true
+          else
+            EXISTS=false
+          fi
+          echo ::set-output name=release-branch::${EXISTS}
```

Comment:
I believe this needs to be an env var

Reply:
outputs set this way only work across jobs IIUC

Reply:
I found other examples of this in the docs.

```
      - name: Set selected color
        run: echo '::set-output name=SELECTED_COLOR::green'
        id: random-color-generator
      - name: Get color
        run: echo "The selected color is ${{ steps.random-color-generator.outputs.SELECTED_COLOR }}"
```

https://docs.github.com/en/free-pro-team@latest/actions/reference/workflow-commands-for-github-actions

Reply:
well then I wonder what I screwed up (or if github quietly fixed it) 🤔